src/userflow.ts

```diff
@@ -357,6 +359,7 @@ if (!userflow) {
   stubVoid('setLinkUrlDecorator')
   stubVoid('openResourceCenter')
   stubVoid('toggleResourceCenter')
+  stubVoid('debugger')
```

Comment:
Although `mountDebugger` inside is `async` task. We need to stub it as sync. Or else, it will block to stub the rest of the methods from sdk and be a hinderance to the actual sdk loading sequence. Because debugger basically loads all the UI deps and stuff.